build.sh

```diff
@@ -34,34 +34,38 @@ dotnet restore ./src/Castle.Services.Logging.log4netIntegration/Castle.Services.
 dotnet restore ./src/Castle.Services.Logging.NLogIntegration/Castle.Services.Logging.NLogIntegration.csproj
 dotnet restore ./src/Castle.Services.Logging.SerilogIntegration/Castle.Services.Logging.SerilogIntegration.csproj
 dotnet restore ./src/Castle.Core.Tests/Castle.Core.Tests.csproj
+dotnet restore ./src/Castle.Core.Tests.NonSigned/Castle.Core.Tests.NonSigned.csproj
 
 # Linux/Darwin
 OSNAME=$(uname -s)
 echo "OSNAME: $OSNAME"
 
 dotnet build ./src/Castle.Core.Tests/Castle.Core.Tests.csproj /p:Configuration=Release || exit 1
+dotnet build ./src/Castle.Core.Tests.NonSigned/Castle.Core.Tests.NonSigned.csproj /p:Configuration=Release || exit 1
 
 echo --------------------
 echo Running NET461 Tests
 echo --------------------
 
 ./src/Castle.Core.Tests/bin/Release/net461/Castle.Core.Tests.exe --result=DesktopClrTestResults.xml;format=nunit3
+./src/Castle.Core.Tests.NonSigned/bin/Release/net461/Castle.Core.Tests.NonSigned.exe --result=DesktopClrNonSignedTestResults.xml;format=nunit3
 
 echo ---------------------------
 echo Running NETCOREAPP1.1 Tests
 echo ---------------------------
 
 dotnet ./src/Castle.Core.Tests/bin/Release/netcoreapp1.1/Castle.Core.Tests.dll --result=NetCoreClrTestResults.xml;format=nunit3
+dotnet ./src/Castle.Core.Tests.NonSigned/bin/Release/netcoreapp1.1/Castle.Core.Tests.NonSigned.dll --result=NetCoreClrNonSignedTestResults.xml;format=nunit3
 
 # Unit test failure
-NETCORE_FAILCOUNT=$(grep -F "One or more child tests had errors" NetCoreClrTestResults.xml | wc -l)
+NETCORE_FAILCOUNT=$(grep -F "One or more child tests had errors" NetCoreClrTestResults.xml NetCoreClrNonSignedTestResults.xml | wc -l)
```

Comment:
Can you confirm that this `grep` call syntax (mentioning several space-separated file names) is correct and will work correctly in the case of a failed unit test?